components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -74,227 +159,556 @@ You can see the actions contained in the `editor` role in the [Roles section]({{
- ]
-}
-```
+### Updating a Policy
 
-To update your policy, replace the last component of the path in the command with the ID of the target policy.
+- You can modify the *membership* of both Chef-managed and custom policies; see [Policy Membership]({{< relref "iam-v2-api-reference.md#policy-membership" >}}) for more information.
+- You can modify the *definition* of custom policies but not Chef-managed policies.
+- The policy ID is immutable; it can only be set at creation time.
+- The update operation modifies *all* policy properties, not just the ones you specify.
+  Properties that you do not specify are reset to empty values.
+  Thus, you should supply all of a policy's properties, not just the ones you wish to update.
 
-```bash
-curl -s -H "api-token: $TOKEN" -d @policy.json -X PUT https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}?pretty
-```
+### Deleting a Policy
 
-You can also make the change specifying the data inline:
+Deleting a policy is permanent and cannot be undone.
 
-```bash
-curl -s -H "api-token: $TOKEN" -d '{"name":"Test Policy", "id":"test-policy", "members":["team:local:alpha","team:ldap:beta"], "statements": [{"actions": [ "iam:users:list"], "effect": "ALLOW"}]}' -X PUT https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}?pretty
-```
+*In the browser:* **Settings**  >> **Policies** >> [open control menu for target policy] >> Delete
 
-### Policy Membership
+## Policy Membership
 
 Chef Automate supports two methods for updating policy membership.
 
-Using PUT lets you **set** membership for the entire policy; see [Setting Policy Membership]({{< relref "iam-v2-api-reference.md#setting-policy-membership" >}}).
+Using `PUT` lets you set membership for the entire policy; see [Setting Policy Membership]({{< relref "iam-v2-api-reference.md#setting-policy-membership" >}}).
 
-Using POST lets you add members to (or remove members from) an existing membership list.
+Using `POST` lets you add members to (or remove members from) an existing membership list.
 For more information, see [Adding Members to Policies]({{< relref "iam-v2-api-reference.md#adding-members-to-policies" >}}) and [Removing Members from Policies]({{< relref "iam-v2-api-reference.md#removing-members-from-policies" >}}).
 
-#### Setting Policy Membership
+Method | HTTP request                           | Description
+-------|----------------------------------------|------------
+list   | GET /policies/***id***/members         | lists the membership
+update | PUT /policies/***id***/members         | replaces the membership
+add    | POST /policies/***id***/members:add    | add to the membership
+remove | POST /policies/***id***/members:remove | remove from the membership
 
-From the command line, specifying a complete list of members is often simpler than keeping track of a policy's membership and adding or deleting specific members (whether users, team, or tokens).
-This command *replaces* the policy's membership.
```

Comment:
I liked the deleted warning: `This command *replaces* the policy's membership.`

Reply:
Again the diff is getting in the way: that is retained under `### Updating All Members on a Policy`